src/components/LandlordDashboard/index.jsx

```diff
@@ -3,7 +3,7 @@ import SLandlordDashboard from './style';
 export default function LandlordDashboard() {
   return (
     <SLandlordDashboard>
-      <p>LandlordDashboard</p>
+      <div>hi</div>
     </SLandlordDashboard>
   );
 }
```

Comment:
? 🤣🤣

Reply:
Waiting for landlord dashboard to be merge

Reply:
The old one seemed better to me